packages/dashboard/src/features/database/pages/IndexesPage.tsx

```
@@ -11,13 +33,31 @@ import {
 import { DatabaseSchemaSelect } from '#features/database/components/DatabaseSchemaSelect';
 import { DEFAULT_DATABASE_SCHEMA, getDatabaseSchemaInfo } from '#features/database/helpers';
+import { useTables } from '#features/database/hooks/useTables';
+import { useConfirm } from '#lib/hooks/useConfirm';
+import { useToast } from '#lib/hooks/useToast';
+
+// Must match backend IDENTIFIER_REGEX
+// eslint-disable-next-line no-control-regex
+const IDENTIFIER_REGEX = /^[^"\x00-\x1F\x7F]+$/;
+
+function validateIdentifierFE(value: string): string | null {
+  if (!value.trim()) return 'Name cannot be empty.';
```

ESLint

[error] Expected { after 'if' condition.

(curly)